include/bluetooth/l2cap.h

```diff
@@ -240,6 +240,9 @@ struct bt_l2cap_server {
 	 *  @param chan Pointer to received the allocated channel
 	 *
 	 *  @return 0 in case of success or negative value in case of error.
+	 *  Possible return values:
+	 *  -ENOMEM if no available space for new channel.
+	 *  -EACCES or -EPERM in case of security reasons.
```

Comment:
You should probably state what EPERM and EACCES map in terms of errors, we could perhaps use EKEYREJECTED instead for the key size but that is not very common error but it is not common to have the a channel requiring a specific key size either.

Reply:
I agree with you. I don't understand why key size and authorization is mandatory for qualification. IMO those should be optional.